common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -636,7 +661,7 @@ def get_content_titles(self):
         This overwrites the get_content_titles method included in x_module by default.
         """
         titles = []
-        for child in self._xmodule.get_child_descriptors():
+        for child in self.get_child_descriptors():
```

Comment:
To keep diff size minimal for now haven't updated descriptor/module variable names. Will do later once the conversions are done.